game_eggs/steamcmd_servers/core_keeper/egg-core-keeper.json

```diff
@@ -0,0 +1,162 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-01-13T12:30:55+01:00",
+    "name": "Core Keeper",
+    "author": "karsten@fiedleronline.net",
+    "description": "Core Keeper is a survival sandbox game.\r\n\r\n--- Drawn towards a mysterious relic, you are an explorer who awakens in an ancient cavern of creatures, resources and trinkets. Trapped deep underground will your survival skills be up to the task? Mine relics and resources to build your base, craft new equipment, survive, and power up the Core. ---\r\n\r\nInfo: Query port ist server port + 1.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:wine_latest": "ghcr.io\/parkervcp\/yolks:wine_latest"
```

Comment:
please use wine staging as if anything changes in the future to the game binary wine staging will likely handle it better

Reply:
Well, I use Wine because of the pre-installed Xvfb. Haven't found a dedicated Xfvb image and don't know if it's absolutely necessary to create one. Wine is definitely not required.

Fixed it now on wine_staging.